### DISCUSSION TITLE
Heur.BZC.ZFV.Boxter.830.95D6E088 detected by Bitdefender.

### DISCUSSION THREAD

**Discussion by AgentNappi007** [AveYo/MediaCreationTool.bat#176] [Q&A]
Bitdefender is giving a warning when executing MediaCreationTool.bat.

Virus waring about powershell is trying to load a bad recource Heur.BZC.ZFV.Boxter.830.95D6E088.

Any Comment to this from the developer ?

**Comment by AveYo** [comment]
"Heur" = heuristic, computer-generated match for similar stuff. 
It's a plain-text script, the language has limited keywords, it's bound to happen that some will be also used by pesky scripts.
**Kindly submit a FALSE POSITIVE report to Bitdefender** with the script and the project url, so that a person can adjust the automatic detection. 

[Bitdefender - Incorrect Detection: Report a False Positive](https://www.bitdefender.com/consumer/support/answer/29358/)

**Reply by AgentNappi007** [reply]
Hi AveYou,
thanks for you fast reply. 
Very much appreciated. 
I've just reported this as a potential false/positive to Bitdefender and they will hopefully quickly fix this. 
Will post ther outcome here in case I'll get any feedback from them.
Thanks and happy coding 🖖🏻
Cheers AgentNappi007

**Comment by joey-like-A-kangaroo** [comment]
Thank you for the post, I also use BD did a full scan and found Heur.BZC.PZQ.Boxter.---------- files in a zip folder I got from Black Hat this year. Good to know it is not actually adware.

**Comment by alandsidel** [comment]
Just want to say this is still happening today, 3 years later.

**Comment by jcalano** [comment]
Still happening.
PowerShell tried to load a malicious resource detected as Heur.BZC.ZFV.Boxter.834.F9814ACC and was blocked
Jan 18, 2026
